http-requests.md

```diff
@@ -39,7 +39,11 @@ TBD
 
 ### User-agent
 
-TBD
+Default value is 'curl/7.54.1', as in 'User-Agent: curl/7.54.1'. Change that for your program's version.
```

Comment:
You think this should perhaps mention that this is just an example as "7.54.1" is the specific version you're talking about? Users are likely to read the book while using a large variety of versions...